libraries/cmake/source/gflags/README.md

```diff
@@ -0,0 +1,61 @@
+# gflags
+
```

Comment:
Can you leave a short one or two sentence description of what the following steps are doing? Same for the `glog` `readme.md`.